kgforge/core/forge.py

```diff
@@ -936,6 +1011,34 @@ def get_model_context(self):
         """Expose the context used in the model."""
         return self._model.context()
     
+    def create_db_sources(self, all_config: Optional[Dict[str, Dict[str, str]]], 
+                          store_config : Optional[Dict[str, Dict[str, str]]],
+                          model_context: Context
+                          ) -> Union[Database, List[Database]]:
+        names = all_config.keys()
+        dbs = {}
+        for name in names:
```

Comment:
you can also iterate over names with .items()
`for name, config in all_config.items()`